pkgs/development/python-modules/pandoc-attributes/default.nix

```diff
@@ -0,0 +1,25 @@
+{ buildPythonPackage
+, fetchPypi
+, pandocfilters
+, stdenv
+}:
+
+buildPythonPackage rec {
+  pname = "pandoc-attributes";
+  version = "0.1.7";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "69221502dac74f5df1317011ce62c85a83eef5da3b71c63b1908e98224304a8c";
+  };
+
+  buildInputs = [
+    pandocfilters
+  ];
+
+  meta = {
+    homepage = https://github.com/aaren/pandoc-attributes;
+    description = "An Attribute class to be used with pandocfilters";
+    license = stdenv.lib.licenses.bsd2;
+  };
```

Comment:
same as above, any reason you don't want to maintain this package? (otherwise you can set a `maintainers` field in `meta`).